config/jobs/kubernetes/sig-network/dualstack-e2e.yaml

```diff
@@ -143,15 +143,15 @@ periodics:
       - --aksengine-template-url=https://raw.githubusercontent.com/aramase/manifests/main/dualstack/ipvs-kubenet.json
```

Comment:
just curious, why is the template name`ipvs-kubenet` for both iptables and ipvs jobs?

Reply:
`1.18`, `1.19` and `1.20` are running with `ipvs-kubenet` templates. The master CI jobs have `iptables` and `ipvs`.

Reply:
ahh, right, the github diff was collapsing some lines and I got confused